services/web-app/components/page-tabs/page-tabs.js

```diff
@@ -6,18 +6,28 @@
  */
 import { Column, Grid, Tab, TabList, Tabs } from '@carbon/react'
 import clsx from 'clsx'
+import { useRouter } from 'next/router'
 import PropTypes from 'prop-types'
+import slugify from 'slugify'
 
 import styles from './page-tabs.module.scss'
 
 const PageTabs = ({ className, tabs = [] }) => {
+  const router = useRouter()
+
+  const currTabSlug = router.asPath.split('/').pop()
```

Comment:
I think we have to do something like this to take the query string out if there's any
```suggestion
  const currTabSlug = router.asPath.split('/').pop().split("?")[0]
```
```suggestion
  const currTabSlug = router.asPath.split('/').pop()
```

Reply:
Ooo okay!